src/ocean/util/log/Logger.d

```diff
@@ -988,7 +988,7 @@ unittest
     const TestStr = "Ce qui se conçoit bien s'énonce clairement - Et les mots pour le dire arrivent aisément";
     scope appender = new Buffer();
     char[32] log_buffer;
-    Logger log = Log.lookup("ocean.util.log.Logger.TestLogTrim")
+    Logger log = (new Logger(Log.hierarchy(), "dummy"))
```

Comment:
We could make it `scope` then

Reply:
AFAICS there is not destructor for `Logger` so it shouldn't matter